covalent/_results_manager/result.py

```diff
@@ -109,71 +109,71 @@ def __str__(self):
         return show_result_str
 
     @property
-    def start_time(self):
+    def start_time(self) -> datetime:
         """
         Start time of processing the dispatch.
         """
 
         return self._start_time
 
     @property
-    def end_time(self):
+    def end_time(self) -> datetime:
         """
         End time of processing the dispatch.
         """
 
         return self._end_time
 
     @property
-    def results_dir(self):
+    def results_dir(self) -> str:
         """
         Results directory used to save this result object.
         """
 
         return self._results_dir
 
     @property
-    def lattice(self):
+    def lattice(self) -> Lattice:
         """
         "Lattice" object which was dispatched.
         """
 
         return self._lattice
 
     @property
-    def dispatch_id(self):
+    def dispatch_id(self) -> str:
         """
         Dispatch id of current dispatch.
         """
 
         return self._dispatch_id
 
     @property
-    def status(self):
+    def status(self) -> Status:
         """
         Status of current dispatch.
         """
 
         return self._status
 
     @property
-    def result(self):
+    def result(self) -> Union[int, float]:
```

Comment:
It might also take the types list or perhaps even dict. Could you double check and add that in?